ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Nagios_Master_Config/tasks/main.yml

```diff
@@ -17,7 +17,7 @@
 # Script must exist on Nagios Master
 #
 - name: SSH into the Nagios Master and excute the Nagios_Ansible_Config_tool.sh script
-  local_action: command ssh -o StrictHostKeyChecking=no root@{{ Nagios_Master_IP }} "/usr/local/nagios/Nagios_Ansible_Config_tool/Nagios_Ansible_Config_tool.sh  {{ ansible_distribution }} {{ ansible_architecture }} {{ inventory_hostname }} {{ ansible_default_ipv4.address }} "
+  local_action: command ssh -o StrictHostKeyChecking=no root@{{ Nagios_Master_IP }} "/usr/local/nagios/Nagios_Ansible_Config_tool/Nagios_Ansible_Config_tool.sh  {{ ansible_distribution }} {{ ansible_architecture }} {{ inventory_hostname }} {{ ansible_host }} "
```

Comment:
Nagios will require the IP address of the client machine
Which out it, Nagios will not be able to find the client machine by unresolvable hostnames 
``` abc123.adoptopenjdk.net ```

Reply:
ansible_host is the IP address that is set in the yml inventory :)